ch5/divide.zig

```diff
@@ -0,0 +1,6 @@
+fn divide(a: i32, b: i32) ?i32 {
```

Comment:
This file is not compiling because it does not have a main function and others files in this PR have the same ""problem"". Maybe adding it with an input example will be nice

WDYT?

Reply:
I'd need to give this extra thought, but the intention is to depict the snippet rather than make it executable.